### DISCUSSION TITLE
Searches by an author should not include pinned discussions unless they're a match

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44024] [Discussions]
### Select Topic Area

Bug

### Body

https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile

My search results are not above the fold (this is a fairly sizable display!):
![image](https://user-images.githubusercontent.com/2119212/211930390-879a1d4a-6839-4ec1-ac51-920fae35b601.png)

It's ok to show pinned results as part of the pre-filing process, but if I'm doing a search, the odds are that I've already seen or intentionally skipped the pinned results and therefore I shouldn't be shown them again, certainly not expanded.

If you want to spend one row per pinned item (shortening to the first Emoji or Word up to 7 chars), that'd be ok:

> [Introducing Projects on GitHub Mobile! (public beta)](https://github.com/orgs/community/discussions/35791) [👂](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3A%22%F0%9F%91%82+Feedback+Wanted%22) [📣](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3A%22%F0%9F%93%A3+ANNOUNCEMENT%22) [Beta](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3ABeta) [Product](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3A%22Product+Feedback%22)
> [Introducing File editing on GitHub Android! (public beta)](https://github.com/orgs/community/discussions/40852) [👂](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3A%22%F0%9F%91%82+Feedback+Wanted%22)[📣](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3A%22%F0%9F%93%A3+ANNOUNCEMENT%22) [Beta](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3ABeta) [Mobile](https://github.com/orgs/community/discussions/categories/mobile?discussions_q=author%3Ajsoref+category%3AMobile+label%3AMobile)

But the fold should not hide the beginning of the user's search results.

**Comment by foxt** [comment]
+1 got confused when I was searching and the results didn't change

<img width="1553" alt="Firefox 2024-10-08 15 08 26" src="https://github.com/user-attachments/assets/4baa7c96-64b6-407b-a2c0-f4fd7ade6898">
